mne/viz/topo.py

```diff
@@ -721,5 +724,7 @@ def plot_topo_image_epochs(epochs, layout=None, sigma=0., vmin=None,
                      fig_facecolor=fig_facecolor, font_color=font_color,
                      border=border, x_label='Time (s)', y_label='Epoch',
                      unified=True, img=True)
+    if fig_background is not None:
+        add_background_image(fig, fig_background)
```

Comment:
to save some work, do the `if fig_background is not None:` check in `add_background_image`, i.e. do nothing if it's `None`, do something if it's not. That way these functions can just make a call instead of each of them doing the same logic